### PR TITLE
VPS-45/daisy-buttons

### PR DESCRIPTION
**Pull request by itsatulbox**
## Describe the issue

The buttons in the repository were the old ugly material ui ones and we want to change them into the nice new daisy ui/tailwind ones.

## Describe the solution

Switching all the buttons to the daisy ui ones and making the widths and functionality the same

## Risk

Some functionality might have been missed, might still look a bit ugly

## Definition of Done

- [ ] Code peer-reviewed
- [ ] Wiki Documentation is written and up to date
- [ ] Unit tests written and passing
- [ ] Integration tests written and passing
- [ ] Continuous Integration build passing
- [ ] Acceptance criteria met
- [ ] Deployed to production environment

## Reviewed By

Who reviewed your PR - for commit history once merged

### CODE REVIEW

File: frontend/src/features/authoring/components/ButtonComponent.jsx [hunk 1]
```diff
@@ -15,15 +15,23 @@ export default function ButtonComponent({ id, onClick, component }) {
   const styles = useStyles(component);
 
   return (
-    <Button
-      style={{ zIndex: component?.zPosition || 0 }}
+    // <Button
```
Comment (wjin-lee) [inline frontend/src/features/authoring/components/ButtonComponent.jsx:18]: potentially unusued?

File: frontend/src/index.css [hunk 2]
```diff
@@ -21,3 +21,15 @@ body,
...
 }
+
+.btn.vps {
+  @apply bg-primary text-secondary hover:opacity-85;
+}
+
+.btn.error {
```
Comment (harbassan) [inline frontend/src/index.css:29]: maybe don't call this error, since its not being used that way. `important` maybe?

**Review by harbassan — APPROVED** [review]

File: frontend/src/features/playScenario/components/PlayPageSideButton/PlayPageSideButton.jsx [hunk 3]
```diff
@@ -14,14 +14,35 @@ export default function PlayPageSideButton({
...
+        <li>
+          <a className="tooltip tooltip-right" data-tip="Notes">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              className="h-8 w-8"
+              fill="black"
+              viewBox="0 0 24 24"
+              stroke="currentColor"
+            >
+              <path d="M.75,17.5A.751.751,0,0,1,0,16.75V12.569a.755.755,0,0,1,.22-.53L11.461.8a2.72,2.72,0,0,1,3.848,0L16.7,2.191a2.72,2.72,0,0,1,0,3.848L5.462,17.28a.747.747,0,0,1-.531.22ZM1.5,12.879V16h3.12l7.91-7.91L9.41,4.97ZM13.591,7.03l2.051-2.051a1.223,1.223,0,0,0,0-1.727L14.249,1.858a1.222,1.222,0,0,0-1.727,0L10.47,3.91Z" />
```
Comment (wjin-lee) [inline frontend/src/features/playScenario/components/PlayPageSideButton/PlayPageSideButton.jsx:28]: We can use MUI icons here!
https://mui.com/material-ui/material-icons/

**Review by wjin-lee — CHANGES_REQUESTED** [review]
Nice job overall!
Just a few nits :)

**Review by wjin-lee — APPROVED** [review]
lgtm!

**Pull request merged**